server/server.go

```diff
@@ -726,11 +716,29 @@ func addEntry(r *rib.RIB, ni string, op *spb.AFTOperation, fibACK bool, election
 		okACK = spb.AFTResult_FIB_PROGRAMMED
 	}
 
-	oks, faileds, err := r.AddEntry(ni, op)
+	var (
+		oks, faileds []*rib.OpResult
+	)
+
+	switch op.Op {
+	case spb.AFTOperation_ADD:
+		oks, faileds, err = r.AddEntry(ni, op)
+	case spb.AFTOperation_DELETE:
+		oks, faileds, err = r.DeleteEntry(ni, op)
+	//case spb.AFTOperation_REPLACE:
```

Comment:
meant to leave this commented out?

Reply:
Yes, but now removed. I haven't yet implemented `REPLACE`.